iOS Client/TabController/MBTabBarController.swift

```diff
@@ -35,15 +76,19 @@ class MBTabBarController: UITabBarController {
     func select(tab: Tab) {
         self.selectedIndex = tab.rawValue
     }
-
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destinationViewController.
-        // Pass the selected object to the new view controller.
+    
+    func newState(state: MBAppState) {
+        if state.podcastsState.player != .initialized {
+            playPauseView?.isHidden = false
+        }
+        
```

Comment:
Nice, do you think this should be hidden on article and/or devotion detail?

Reply:
I could envision someone just reading a devotion or article with a podcast in the background and then want to pause it. Open for debate though